src/js/addons/widgets/Widget.js

```diff
@@ -1,34 +1,62 @@
+/**
+ *  A widget is a standalone unit of functionality for a
+ *  given matching context in Tangram YAML.
+ *
+ *  An instance of the widget class stores a reference to
+ *  its DOM element (and children) and can be extended
+ *  with any additional functionality.
+ *
+ */
+
+import { getPosition } from '../../core/common.js';
+
+// Store reference to the editor on this module
+let editor;
```

Comment:
let's remove globals on modules as much as we can specially if their name repites with global variables. There is a window.editor defined on TangramPlay play for debugging porposes